Build/net45.props

```diff
@@ -10,14 +10,11 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <Features>FEATURE_CUSTOM_MODIFIERS</Features>
     <Features>$(Features);FEATURE_NATIVE</Features>
```

Comment:
I thought about that, but then I saw that net40.props didn't do it so I left it as is. If you think it'll cause issues I can change it.